src/Core/Users/classUser.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Meetup\Login\Core;
+use Meetup\src\Core\Users\UserInterface
+Class User  implements  UserInterface{
+private $firstName;  
+private $lastName;  
+private $createdAt;  
+private $updatedAt;  
+private $email;  
+private $password;  
+
+	public function __construct($email, $password){
+		$this->setEmail($email);
+		$this->setPassword($password);
+	}
+	public function setPassword($password){
+		$this->password = $password;
```

Comment:
Please read the interface docs. The password should be hashed before assigning to a variable.